src/components/core/toast.scss

```
@@ -0,0 +1,39 @@
+.ctw-toast-override {
+  --toastify-color-info: #4b5563 !important;
+  --toastify-z-index: 10001 !important; /* one higher than the drawer */
```

Modal's are even higher at `10100` (from `modal.tsx`). We only use modal for confirm delete prompt and for the CCDA modal.

```suggestion
  --toastify-z-index: 10101 !important; /* one higher than modals which are above drawers */
```